manifests/database/postgresql.pp

```diff
@@ -7,11 +7,19 @@
 
   include postgresql::client, postgresql::server
 
+  if $facts['os']['family'] == 'RedHat' {
+    stdlib::ensure_packages(['glibc-langpack-en'])
+    $require_langpack = Package['glibc-langpack-en']
+  } else {
+    $require_langpack = undef
```

Comment:
I think this also works and is shorter:
```suggestion
    Package['glibc-langpack-en'] -> Postgresql::Server::Db[$foreman::db_database]
```

Then you can leave out the `require` below